utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1

```diff
@@ -76,7 +76,7 @@ function Convert-TokenInFile {
             if ($TokenNameValueObject[$_] -is [System.Security.SecureString]) {
                 $TokenNameValueObject[$_] = $TokenNameValueObject[$_] | ConvertFrom-SecureString -AsPlainText
             }
-            $File = $File -replace $_, $TokenNameValueObject[$_]
+            $File = $File -replace [Regex]::Escape($_), $TokenNameValueObject[$_]
```

Comment:
Is this change intended or coming from previous tries to fix?

Reply:
It is intended. The `[` is a rege character and `-replace` (other than `.replace()`) uses regex